scipy/stats/_fit.py

```diff
@@ -565,3 +566,521 @@ def nllf(free_params, data=data):  # bind data NOW
+
+    Given a distribution family and data, perform a test of the null hypothesis
+    that the data were drawn from a distribution in that family. Any known
+    parameters of the distribution may be specified. Remaining parameters of
+    the distribution will be fit to the data, and the p-value of the test
+    is adjusted accordingly. Several statistics for comparing the distribution
+    to data are available.
+
+    Parameters
+    ----------
+    dist : `scipy.stats.rv_continuous`
+        The object representing the distribution family to be fit to the data.
+    data : 1D array_like
+        Finite, uncensored data to which the distribution is to be fit.
+    known_params : dict, optional
+        A dictionary containing distribution name-value pairs of distribution
+        parameters that are known. Data is resampled from the null-hypothesized
+        distribution with these values of the parameters. Before the statistic
+        is evaluated for each resample, only remaining unknown parameters of
+        the  null-hypothesized distribution family are fit to the resampled
+        data; these parameters are held fixed. (If all parameters of the
+        distribution family are known, then the step of fitting the
+        distribution family to each resample is omitted.)
+    fit_params : dict, optional
+        A dictionary containing name-value pairs of distribution parameters
+        that have already been fit to the data, e.g. using `scipy.stats.fit`
+        or the ``fit`` method of `dist`. Data is resampled from the
+        null-hypothesized distribution with these specified values of the
+        parameters, but these and all other unknown parameters of the
+        null-hypothesized distribution family are fit to the resampled data
+        before the statistic is evaluated.
+    guessed_params : dict, optional
+        A dictionary containing name-value pairs of distribution parameters
+        which are guessed. Before resampling from the null-hypothesized
+        distribution, the null-hypothesized distribution family is fit to the
+        data with these parameters free, but the provided values will be used
+        as guesses if needed by the fitting procedure. The null-hypothesized
+        distribution family is fit to the resampled data with these parameters
+        free before the statistic is evaluated.
+    statistic : {"ad", "ks"}, optional
+        The statistic used to compare data to a distribution after fitting
+        unknown parameters of the distribution family to the data. The
+        Anderson-Darling ("ad") [1]_ and Kolmogorov-Smirnov ("ks") [2]_
+        statistics are available.
+    fit_method : {"mle", "mm"}, optional
+        The method used to fit null-hypothesized distribution family to
+        the resampled data. Maximum likelihood estimation ("mle") is the
+        default; method of moments ("mm") is also available
```

Comment:
```suggestion
        default; method of moments ("mm") is also available.
```
Note that bias-correction procedures may be implemented, so results may differ from those of strict MLE or MoM.